src/pages/chart/chart.scss

```diff
@@ -103,12 +103,17 @@
     }
 
     .smartcharts-desktop {
+        .cq-modal__overlay {
+            position: fixed !important;
+            height: 100vh !important;
+            width: 100% !important;
+            background-color: rgb(0 0 0 / 70%);
+            z-index: 9999 !important;
+        }
```

Comment:
@mayuran-deriv can you please check this in a real mobile device? Using vh sometimes causes issues in mobile due to the browser elements at the top and the bottom.